signals/content/subject_is_upper.yml

```diff
@@ -0,0 +1,4 @@
+name: "Subject: Contains All Capital Letters"
+type: "query"
+source: |
+  regex.match(subject.subject, "^[A-Z ]*$")
```

Comment:
I think we might want to do it this way? 

`^[^a-z]*[A-Z][^a-z]*$`

That way we capture when a subject contains other chars along with ALL CAPS?

like
`#:SCAN ORIGINAL QUOTE SHIPPING DOCS-MSK///2131
`

Reply:
Yeah, that makes sense to me. I wasn't sure if we wanted multiple signals to represent:
- All upper
- Upper w/ symbols
- ... etc